components/Resources.js

```diff
@@ -18,6 +18,14 @@ const resourceCategories = [
         name: "Developer's Guide to Content Creation",
         url: "https://www.stephaniemorillo.co/product-page/the-developer-s-guide-to-content-creation",
       },
+      {
+        name: "Developer Relations: How to Build and Grow a Successful Developer Program",
+        url: "https://www.devrelbook.com/",
+      },
+      {
+        name: "The Developer Advocacy Handbook",
+        url: "https://developer-advocacy.com/",
```

Comment:
I've never heard of this and would rather not add it since I typically like multiple people to recommend a specific resource before it gets added. Could you delete for now please?